### DISCUSSION TITLE
Git commit message 이슈번호 질문 드립니다.

### DISCUSSION THREAD

**Discussion by Hyun-Soon** [lawnmowing-programmer/algo#74] [Q&A] [answered]
Git commit message 작성할 때 #이슈번호는 뭘 기준으로 작성하면 될까요?

**Comment by minjae9610** [comment, marked as answer]
레포지토리에 작성되는 모든 게시글은 순서대로 고유번호가 부여됩니다. 이슈를 작성하시고 보시면 번호가 자동으로 부여되어있을거에요. PR도 마찬가지고요 :)

**Reply by Hyun-Soon** [reply]
감사합니다!
